src/twisted/internet/protocol.py

```diff
@@ -623,6 +623,8 @@ class ProcessProtocol(BaseProtocol):
     stdin, stdout, and stderr file descriptors.
     """
 
+    transport: Optional[interfaces.IProcessTransport] = None
```

Comment:
`ProcessProtocol` code that uses `self.transport` doesn't type check without this.